valo_api/responses/store_featured.py

```diff
@@ -40,6 +40,7 @@ class BundleV1(Struct):
 
 
 class BundleV2(Struct):
+    bundle_price: int
     bundle_uuid: str
```

Comment:
```suggestion
    bundle_uuid: str
    bundle_price: int
```

I would set the uuid first here :D


Just a small change